services/supervisord/supervisord.go

```diff
@@ -558,3 +570,28 @@ stdout_logfile_backups = 3
 redirect_stderr = true
 {{end}}
 `))
+
+// todo (valayla) - what are we going todo with default memory usage and ` -memory.allowedPercent`.
+var victoriaMetricsTemplates = template.Must(template.New("").Option("missingkey=error").Parse(`
+{{define "victoriametrics"}}
+[program:victoriametrics]
+priority = 7
+command =
+	/usr/sbin/victoria-metrics
+		--promscrape.config=/etc/victoriametrics-promscrape.yml
+		--retentionPeriod={{ .DataRetentionMonth }}
+		--storageDataPath=/srv/victoriametrics/data
+		--httpListenAddr=127.0.0.1:8428
+user = pmm
+autorestart = true
+autostart = true
```

Comment:
We should render that file in all cases, but that value, and one above, should be `false` if `isVMEnabled` is false

Reply:
Done, config rendered in all cases.